src/types.ts

```diff
@@ -13,6 +13,11 @@ export interface StoredWallet {
   wallet: string // encrypted wallet
   network: string
 }
+export interface StoredSeed {
+  name: string
+  //email TODO
```

Comment:
why?

Reply:
Well, in the future we will connect users and emails... but yeah, maybe not needed in localStorage. Better for database only